src/styles/variables.scss

```diff
@@ -0,0 +1,15 @@
+$font-family: Arial;
+$day-size: 100px;
+$gap: 1px;
+$padding: 10px;
+$day-font-size: 30px;
+$day-color: #eee;
+$days: (
+  sun: 0,
+  mon: 1,
+  tue: 2,
+  wed: 3,
+  thu: 4,
+  fri: 5,
+  sut: 6,
```

Comment:
It looks like there is a typo in the name of the last day of the week. It should be 'sat' for Saturday instead of 'sut'. Please correct this to ensure the day names are accurate.